partcad-cli/src/partcad_cli/click/commands/convert.py

```
@@ -18,25 +17,25 @@
     """
-    pc_logging.info(f"Starting conversion: '{object_name}' -> '{target_format}', dry_run={dry_run}")
+    pc_logging.info(f"Starting conversion: '{part_name}'.")
 
     project = ctx.get_project("")
     if not project:
         pc_logging.error("Project retrieval failed. Ensure you are inside a valid PartCAD project.")
         raise click.UsageError("Failed to retrieve the project.")
+    try:
+        convert_part_action(project, part_name, target_format, output_dir=output_dir, dry_run=dry_run)
+    except ValueError as e:
+        raise click.UsageError(str(e))
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)